pkg/cloudprovider/cloudprovider.go

```diff
@@ -241,6 +241,23 @@ func (c *CloudProvider) GetSupportedNodeClasses() []status.Object {
 	return []status.Object{&v1alpha2.AKSNodeClass{}}
 }
 
+// TODO: review repair policies
```

Comment:
Is this TODO something you intend to do in this PR? Or later?

Reply:
Later. This one only introduces the two most sensible ones around NodeReady condition, but there could be others. (Node Repair is behind a feature flag BTW, disabled by default).